full data update

ramirezg opened the pull request:
Updated all yaml data. Just wanted to do this before applying any formatting changes.

ramirezg commented on contacts/data/FMC.yaml:
```diff
@@ -13,9 +13,6 @@ departments:
   public_liaison: 'Rachel Dickon, Phone: (202) 523-5725'
-  reading_rooms:
-  - - Electronic Reading Room
-    - http://www.fmc.gov/electronic_reading_room/default1.aspx
```
Looks like we lost a reading room in the update. We could add this to the manual data.

konklone replied:
:+1:

konklone commented on contacts/data/DOJ.yaml:
```diff
@@ -2789,8 +2793,6 @@ departments:
   - - FOIA Library
     - http://www.justice.gov/usao/resources/foia-library
-  - - FOIA Library
-    - http://www.justice.gov/usao/foia-library
```
Looks legitimately removed.

konklone commented on contacts/data/DoD.yaml:
```diff
@@ -227,7 +232,7 @@ departments:
       simple_median_days: '8'
   service_center: 'Phone: (703) 696-2735'
-  top_level: true
+  top_level: false
```
Should this be switched to `false`?

ramirezg replied:
Nope, good catch! It looks like I accidentally added another `Department of the Air Force - Headquarters/ICIO (FOIA)` while updating the emails.

konklone commented on contacts/data/GSA.yaml:
```diff
@@ -169,7 +170,6 @@ keywords:
 - government forms
 - purchase cards
 name: General Services Administration
-request_form: https://foiaonline.regulations.gov/foia/action/public/home
```
I wonder why GSA removed this from their contact data on FOIA.gov?

ramirezg replied:
It's still under [Headquarters](https://github.com/18F/foia/blob/update_yamls/contacts/manual_data/GSA.yaml#L6-L8), but we could move it over to the top-level as well. Thoughts?

khandelwal replied:
The fewer manual changes we make, the better. So if it moved, it moved.

konklone commented on contacts/data/NTSB.yaml:
```diff
@@ -14,9 +14,6 @@ departments:
   public_liaison: 'Melba D. Moye, Phone: (202) 314-6540'
-  reading_rooms:
-  - - Electronic Reading Room
-    - http://www.ntsb.gov/foia/foia_err.html
```
Looks like they moved it here: http://www.ntsb.gov/about/foia/Pages/foia_err.aspx

I think we should definitely add it to our manual data, since the FOIA.gov site no longer has any reading room listed.

khandelwal replied:
This one is interesting. The reading room link doesn't get picked up because the FOIA page for NTSB is now broken. Our dataset has: http://www.ntsb.gov/foia/index.html. Since that page doesn't work, there's no where for the reading rooms scraper to go get a reading room link. 

Why is the FOIA page link broken?

(Note foia.gov doesn't list reading rooms)

konklone commented on contacts/data/PC.yaml:
```diff
@@ -134,7 +134,6 @@ keywords:
 - Reporting and recordkeeping requirements
 - Technical assistance
 name: Peace Corps
-request_form: https://cwppc.secureprtportal.com/
```
http://www.peacecorps.gov/about/policies/foia/ still links to https://cwppc.secureprtportal.com/ as their online request form.

ramirezg replied:
The form url still under `Main Office` and `OIG` this is the same issue as GSA.

konklone commented on contacts/data/US RRB.yaml:
```diff
@@ -160,5 +160,4 @@ keywords:
 - Vocational rehabilitation
 - Wages
 name: Railroad Retirement Board
-request_form: https://secure.rrb.gov/efoia/
```
http://www.rrb.gov/blaw/foia/foia.asp still links to https://secure.rrb.gov/efoia/ as their online FOIA form.

konklone commented on contacts/data/USDA.yaml:
```diff
@@ -969,9 +1014,6 @@ departments:
   public_liaison: 'Albert Conerly, Phone: (202) 720-0234'
-  reading_rooms:
-  - - FOIA Reading Room
-    - http://www.gipsa.usda.gov/foia.html
```
USDA lists all the agency reading rooms here: http://www.dm.usda.gov/foia/agencyfoia.htm and it includes the one for GIPSA - but it is a 404. Not sure what's going on.

khandelwal replied:
That's why it got removed from the dataset - because it 404s.

konklone commented on contacts/data/VA.yaml:
```diff
@@ -1568,37 +1598,47 @@ keywords:
 public_liaison: 'Phone: (202) 632-7600'
-reading_rooms:
-- - FOIA Library
-  - http://www.oprm.va.gov/foia
```
This whole website is down, and I can't figure out why.

ramirezg replied:
It's up now, but either way this isn't a reading room.

ramirezg commented:
Updates:
1. Cleaned up the [manual data tests](https://github.com/18F/foia/blob/update_yamls/contacts/tests/test_manual_data.py) and added checking for duplicates. 
2. Corrected the issues in the comments, mostly by changing manual data.

ramirezg commented on contacts/data/DOC.yaml:
```diff
@@ -35,8 +35,6 @@ departments:
   public_liaison: 'Jennifer Goode, PH.D., , Phone: (301) 763-2127'
   reading_rooms:
   - - FOIA Library
-    - http://www.census.gov/foia/foia_library/
```
Maybe add this one in manually. It still seems to work.

ramirezg commented on contacts/data/DOC.yaml:
```diff
@@ -221,7 +219,8 @@ departments:
   - Washington, DC 20230
-  emails: []
+  emails:
+  - SKong@eda.gov
```
This is correct, it's been fixed.

ramirezg commented on contacts/data/DOE.yaml:
```diff
@@ -651,8 +651,6 @@ departments:
   public_liaison: 'Derek Passarelli, Phone: (720) 356-1742'
   reading_rooms:
   - - Golden Field Office Reading Room
-    - http://energy.gov/eere/office-energy-efficiency-renewable-energy
```
Update is correct.

ramirezg commented on contacts/data/DoD.yaml:
```diff
@@ -360,7 +366,7 @@ departments:
       simple_lowest_days: less than 1
   service_center: 'Phone: (703) 428-6238'
-  top_level: false
+  top_level: true
```
This is fixed now, there was a duplicate in the manual data. The updated tests should check for issues like this now.

ramirezg commented on contacts/data/NASA.yaml:
```diff
@@ -963,7 +963,7 @@ departments:
       simple_median_days: '8'
   service_center: 'Phone: (228) 813-6174'
-  top_level: false
+  top_level: true
```
Manual data fixed.

ramirezg commented on contacts/data/USDA.yaml:
```diff
@@ -243,8 +244,6 @@ departments:
     - http://www.aphis.usda.gov/wps/portal/aphis/resources/foia?1dmy&urile=wcm%3apath%3a%2Faphis_content_library%2Fsa_resources%2Fsa_laws_and_regulations%2Fsa_foia%2Fct_foia_reading_room
   - - Freedom of Information Act (FOIA) Reading Room
-    - http://www.aphis.usda.gov/wps/portal/aphis/resources?1dmy&urile=wcm%3apath%3a/aphis_content_library/sa_resources/sa_laws_and_regulations/sa_foia/ct_foia
-  - - Freedom of Information Act (FOIA) Reading Room
```
still have 2 links that work.

ramirezg commented on contacts/manual_data/DoD.yaml:
```diff
@@ -53,8 +55,6 @@ departments:
   emails: []
   website: http://www.asbca.mil
-- name: Department of the Army - Freedom of Information and Privacy Office
-  request_form: https://www.rmda.army.mil/foia/foiarequest/default.aspx
```
Moved like GSA's request_form

ramirezg commented on contacts/manual_data/NASA.yaml:
```diff
@@ -27,8 +28,6 @@ departments:
 - name: Langley Research Center
   request_form: http://socialforms.nasa.gov/foia
-- name: NASA Shared Service Center (NSSC)
-  request_form: http://socialforms.nasa.gov/foia
```
deleted duplicate.

ramirezg commented on contacts/manual_data/USPS.yaml:
```diff
@@ -12,5 +13,3 @@ departments:
 - name: Office of the Inspector General
   website: https://uspsoig.gov/general/foia
-- name: USPS Headquarters Office
-  request_form: https://pfoiapal.usps.com/palMain.aspx
```
deleted duplicate.

ramirezg commented on contacts/manual_data/DoD.yaml:
```diff
@@ -66,5 +66,3 @@ departments:
 - name: U.S. Africa Command
   emails:
   - africom.stuttgart.accc.list.foia-mba@mail.mil
-- name: Department of the Air Force - Headquarters/ICIO (FOIA)
```
Deleted duplicate.

ramirezg commented:
Should take care of 18F/foia-hub/issues/407
